react/RecommendationRefresh.tsx

```diff
@@ -34,7 +35,9 @@ const RecommendationRefresh: StorefrontFunctionComponent<Props> = ({
   }, [error, data])
 
   return recommendations ? (
-    <ExtensionPoint id="refresh-shelf" recommendedLists={recommendations} />
+    <RecommendationProvider shouldSendEvents>
```

Comment:
it shoud send if the size of the recomendation is 0?

Reply:
whatever. the `refresh-shelf` component only send the event if the user see the shelf. with no recommendations the shelf is not displayed, so the user don't see it.